From 6f92c6ee1ec282a04c75ef230a10c52df31f5fe1 Mon Sep 17 00:00:00 2001
From: Caesar <wxt@rock-chips.com>
Date: Tue, 3 Apr 2018 20:33:27 +0800
Subject: [PATCH] arm64: dts: rockchip: force the bootargs for rk3399 linux

The default GPT will set the same with android tool.

Address  Name       Path
         Boot     -- rk3399_loader_xx.xx.bin
0x4000   loader1  -- out/u-boot/uboot.img
0x6000   atf      -- out/u-boot/trust.img
0x8000   resource -- out/kernel/resource.img
0x8000   kernel   -- out/kernel/kernel.img
0x40000  rootfs   -- out/rootfs/buildroot.img

Change-Id: I03c4f8221d98f0452928ed21b611db5ff90fc48f
Signed-off-by: Caesar <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
index 32299b2f0844..962098b842df 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-linux.dtsi
@@ -46,6 +46,10 @@
 / {
 	compatible = "rockchip,linux", "rockchip,rk3399";
 
+	chosen {
+		bootargs = "earlycon=uart8250,mmio32,0xff1a0000 swiotlb=1 console=ttyFIQ0 root=PARTUUID=614e0000-0000 rootfstype=ext4 rootwait";
+	};
+
 	reserved-memory {
 		#address-cells = <2>;
 		#size-cells = <2>;
-- 
2.35.3

